app/views/layouts/_header.html.erb

```diff
@@ -1,6 +1,8 @@
 <div class="header">
   <h1 class="title">Check In <i class="fa-solid fa-clock"></i></h1>
-  <button class="primary-button-style">
-    Admin Tools
-  </button>
+    <% if current_page?(:controller => 'admin', :action => 'login') %>
```

Comment:
According to Ruby 3 guide style, is better use shorthand hash literal syntax when all keys are symbols.

`# bad
{ :a => 1, :b => 2 }

# good
{ a: 1, b: 2 }`

Reply:
i aggre with @alexs, can read more about the ruby syntax [here](https://www.rubyguides.com/2019/10/ruby-syntax/)

Reply:
Done, and thanks! @JuanequeX @alexs